app/javascript/src/NewService/components/ServiceDiscoveryForm.jsx

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import React from 'react'
+// TODO: Get authenticityToken from rails
+// Is this needed, or was just generated by formtastic ?
+const authenticityToken = ''
```

Comment:
You could use this: https://github.com/3scale/porta/blob/master/app/javascript/src/Policies/components/CustomPolicy.jsx#L35